src/views/AddPriceMultiple.vue

```diff
@@ -50,18 +50,37 @@
                 :loading="createProofLoading">
               </v-file-input>
               <p v-if="proofFormFilled && !createProofLoading" class="text-green mt-2 mb-2">
-                  <i v-if="!proofSelectedMessage">{{ $t('AddPriceSingle.PriceDetails.ProofUploaded') }}</i>
-                  <i v-if="proofSelectedMessage">{{ $t('AddPriceSingle.PriceDetails.ProofSelected') }}</i>
+                  <i v-if="!proofisSelected">{{ $t('AddPriceSingle.PriceDetails.ProofUploaded') }}</i>
+                  <i v-if="proofisSelected">{{ $t('AddPriceSingle.PriceDetails.ProofSelected') }}</i>
                 </p>
               <p v-if="!proofFormFilled && !createProofLoading" class="text-red mt-2 mb-2">
                 <i>{{ $t('AddPriceSingle.PriceDetails.UploadProof') }}</i>
               </p>
               <p v-if="proofType === 'RECEIPT'" class="text-caption text-warning">
                 <i>{{ $t('AddPriceMultiple.ProofDetails.ReceiptWarning') }}</i>
+                <i>{{ $t('AddPriceMultiple.ProofDetails.PrivateWarning') }}</i>
               </p>
             </v-col>
-            <v-col v-if="proofFormFilled">
+            <v-col  cols="12" sm="12" md="4" v-if="proofFormFilled">
```

Comment:
```suggestion
            <v-col cols="12" md="4" v-if="proofFormFilled">
```

Reply:
also, when switching to md, the whole section switches as well, so there is not that much space for the proof image + toggle
![image](https://github.com/openfoodfacts/open-prices-frontend/assets/7147385/04ed9eaf-70ef-49f0-b84f-5b72d16c55b2)